app/views/clients/clients.html

```diff
@@ -18,6 +24,26 @@
         </tr>
         </thead>
         <tbody>
+         <tr class="pointer-main" ng-repeat="result in searchResults" ng-show="result.entityType  == 'CLIENT'" >
+             <td class="pointer" data-ng-click="routeTo(result.entityId)"> {{result.entityName}} </td>
+             <td class="pointer" data-ng-click="routeTo(result.entityId)"> {{result.entityExternalId || result.entityAccountNo}} </td>
+             <td class="pointer" data-ng-click="routeTo(result.entityId)">
+                 <i class="icon-stop {{result.entityStatus.code | StatusLookup}}"></i>&nbsp;{{result.entityStatus.value}}
+                 <small ng-if="result.entitySubStatus.description">[{{result.entiytSubStatus.description}}]</small></td>
+             <td class="pointer" data-ng-click="routeTo(result.entityId)"> {{result.parentName}} </td>
+         </tr>
+        </tbody>
+
+        <tbody>
+        <tr class="pointer-main" ng-repeat="result in searchResults" ng-show="result.entityType  == 'CLIENTIDENTIFIER'" >
+            <td class="pointer" data-ng-click="routeTo(result.entityId)"> {{result.parentName}} </td>
```

Comment:
entity Id in the case of a ClientIdentifier refers to the entity Id of the Client Identifier itself (and not the client), so you would end up routing to the wrong client here...